src/app/app-routing.module.ts

```diff
@@ -2,11 +2,15 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { LoginComponent } from './login/login.component';
 import { RegisterComponent } from './register/register.component';
+import { RestaurantlistComponent } from './restaurantlist/restaurantlist.component';
+import { NewrestaurantComponent } from './newrestaurant/newrestaurant.component';
 
 const routes: Routes = [
   { path: '', component: LoginComponent },
   { path: 'login', component: LoginComponent },
   { path: 'register', component: RegisterComponent },
+  { path: 'restaurant-list', component: RestaurantlistComponent },
+  { path: 'new-restaurant', component: NewrestaurantComponent },
```

Comment:
It would be better to nest all restaurant routes within the `restaurants` route. That way we can have a domain `restaurants` that will wrap all child routes.
For example:
- `/restaurants`
- `/restaurants/new`
- `/restaurants/:id`
- `/restaurants/:id/comments`

Reply:
I thought it should be a name like a components name. Is there an easy way for me to find where this name is used to avoid forgetting anything?

Reply:
Not really, searching for the sting in project might be best bet